spec/presenters/song_presenter_spec.rb

```diff
@@ -2,7 +2,15 @@
 
 describe 'SongFacade' do
   before :each do
-    @facade = SongFacade.new('token')
+    user = User.create(spotify_id: "",
```

Comment:
This `user` needs to be `@user` as it's being instantiated in each test in this spec.

Reply:
We are using @facade for each test not user, so, that should not be an issue